moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -345,9 +331,6 @@ bool JogCalcs::cartesianJogCalcs(geometry_msgs::TwistStamped& cmd, JogArmShared&
 
   enforceSRDFAccelVelLimits(delta_theta_);
 
-  if (!addJointIncrements(internal_joint_state_, delta_theta_))
```

Comment:
@AndyZe Do you agree that this was redundant? I removed it because it was called again in `convertDeltasToOutgoingCmd()` in the end of this function.

Reply:
Yeah, that seems like a good change